horace_core/Tobyfit/tobyfit_DGfermi_resconv_init.m

```diff
@@ -95,9 +95,9 @@
 %       Cell array of widths of energy bins, one array per dataset
 %           dt          Time widths for each pixel (s), size [npix,1]
 %
-%       Cell arrays of q,w and transformation arrays, one array per dataset
-%           qw          Cell array size [1,4] with components of momentum (in rlu) and energy
-%                       for each pixel [Columns of length npix]
+%       Arrays of w and transformation arrays, one array per dataset
```

Comment:
The line has inherited what was already an erroneous comment about transformation arrays: for consistency with e.g. line 95:
''%    Cell array of energy bin centres, one array per dataset"
Do we want this as a column instead of a row?